src/filters/compress/config.rs

```diff
@@ -21,7 +21,8 @@ use serde::{Deserialize, Serialize};
 
 use super::compressor::{Compressor, Snappy};
 use super::quilkin::filters::compress::v1alpha1::{
-    compress::Action as ProtoAction, compress::Mode as ProtoMode, Compress as ProtoConfig,
+    compress::{Action as ProtoAction, ActionValue, Mode as ProtoMode, ModeValue},
```

Comment:
Should we switch this one out to be `use super::quilkin::filters::compress::v1alpha1 as proto` and not alias each import individually, like has been refactored in the other Filters, for consistency?